athina/steps/llm.py

```
@@ -185,14 +185,15 @@ def simple(
         message: str,
         model: str = Model.GPT4_O.value,
         name: Optional[str] = None,
+        model_options: ModelOptions = ModelOptions(),
```

Ruff (0.8.2)

Do not perform function call `ModelOptions` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)